tests/installation/add_update_test_repo.pm

```diff
@@ -42,6 +43,8 @@ sub run() {
         # if more repos to come, add more
         send_key_until_needlematch('addon-menu-active', 'alt-a', 10, 2) if @repos;
     }
+    wait_still_screen(2);
+    send_key 'alt-b' if check_screen('addon-menu-active');
```

Comment:
It'd be nice to see here what `alt+b` does

Reply:
changed and upodated

reason: https://openqa.suse.de/tests/4871232#step/system_role/2
VR:
https://openqa.suse.de/tests/4871740 s390x
http://10.100.12.155/tests/16764 x86_64